cetmix_tower_aws/readme/CONTEXT.md

```
@@ -0,0 +1,5 @@
+Although Amazon Web Services (AWS) allows API calls without using an SDK, we found that
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)